### PR TITLE
Restrict objdump to relevant sections of elf

### PR DESCRIPTION
**Pull request by dstutt**
An extension of PR#1711

### CODE REVIEW

**Comment by dstutt** [comment]
I decided to make the same changes to all tests that use llvm-objdump

File: llpc/test/shaderdb/general/CsPipelineDumpTest.pipe [hunk 1]
```diff
@@ -46,9 +46,9 @@
 
 ; Compare the disassembly to make sure all 3 Elfs have the same code. We do not expect the binaries to be bit-identical,
 ; but the text section and symbol table should be the same.
-; RUN: llvm-objdump --triple=amdgcn --mcpu=gfx900 --syms --reloc -d %t.orig.elf | tail -n +4 > %t.orig.s
-; RUN: llvm-objdump --triple=amdgcn --mcpu=gfx900 --syms --reloc -d %t/dump/PipelineCs_0x*.elf | tail -n +4 > %t.dump.s
-; RUN: llvm-objdump --triple=amdgcn --mcpu=gfx900 --syms --reloc -d %t.recompile.elf | tail -n +4 > %t.recompile.s
+; RUN: llvm-objdump --triple=amdgcn --mcpu=gfx900 --syms --reloc -d -j .text -j .rodata %t.orig.elf | tail -n +4 > %t.orig.s
```
Comment (jayfoad) [inline llpc/test/shaderdb/general/CsPipelineDumpTest.pipe:49]: Is any change required for tests that were already using `-d` (lower case)? Doesn't that mean something like "only disassemble .text"?

Comment (dstutt) [reply llpc/test/shaderdb/general/CsPipelineDumpTest.pipe:49]: -d means disassemble (all executable sections found in the input files)
-D means disassemble-all (all sections found in the input files)
So maybe? Although using -j make sure.
Perhaps we do want to disassemble all executable sections though.

**Comment by jayfoad** [comment]
My inclination would be to use `-d` for normal tests, and `-d -j ...` (instead of `-D`) only for tests that really want to disassemble .rodata.

**Comment by dstutt** [comment]
Ok - makes sense. See updated commit.

**Review by jayfoad — APPROVED** [review]

**Review by amdrexu — DISMISSED** [review]

**Comment by jayfoad** [comment]
!help

**Review by jayfoad — APPROVED** [review]

**Comment by amdvlk-admin** [comment]
### Test summary for commit 26da2a7b4b20900bcc505e6d09513c2451022f9c
<details><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master 39f8940199e60c44d4211cf8165dfd12876316fa</li>
<li>METROHASH: amd-master 3c566dd9cda44ca7fd97659e0b53ac953f9037d2</li>
<li>PAL: dev 2483d46fa27c30502e497ea169ee53b142e9fa06</li>
<li>SPVGEN: dev 9b30a4a91ec444943b23843853dfce2c6618f8fc</li>
<li>XGL: dev 1ce25b1ed8829c27645edd646a3289e4c524c84c</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev 0d6cb5407b689f465d37b7f39e2de2c747ef28f1</li>
</ul></details>

<details><summary>CTS tests (Failed: 0/203169)</summary><ul><li>Built with version 1.3.0.0</li>
<details open><summary>Rhel 8.2, Gfx10</summary><ul>

<li>Passed: 38237/67723 (56.5%)</li>
<li>Failed: 0/67723 (0.0%)</li>
<li>Not Supported: 29486/67723 (43.5%)</li><li>Warnings: 0/67723 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Passed: 38075/67723 (56.2%)</li>
<li>Failed: 0/67723 (0.0%)</li>
<li>Not Supported: 29648/67723 (43.8%)</li><li>Warnings: 0/67723 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx8</summary><ul>

<li>Passed: 39235/67723 (57.9%)</li>
<li>Failed: 0/67723 (0.0%)</li>
<li>Not Supported: 28488/67723 (42.1%)</li><li>Warnings: 0/67723 (0.0%)</li>
</ul></details>


</ul></details>

**Review by amdrexu — DISMISSED** [review]

**Review by jayfoad — APPROVED** [review]

**Review by amdrexu — APPROVED** [review]

**Pull request merged**